torch/utils/data/datapipes/utils/decoder.py

```diff
@@ -30,6 +26,7 @@ def basichandlers(key, data):
             return None
 
     if extension in "json jsn":
+        import json
```

Comment:
Why import `json` lazily? IIRC it has no system dependencies and is available in the standard library.